src/notifications/use-push-notifications.tsx

```diff
@@ -53,23 +54,30 @@ export const usePushNotifications = () => {
   }, []);
 
   const register = useCallback(async () => {
+    const token = await messaging().getToken();
```

Comment:
This could potentially return an error if `requestPermission` has not been called yet. This _should_ not happen if the onboarding works correctly. But as you know, everything that should not happen happens.

Reply:
I only know what the doc say: "On iOS you'll need to register for remote notifications before calling this method, you can do this by calling registerDeviceForRemoteMessages or requestPermission as part of your app startup. If you have not registered and you call this method you will receive an 'unregistered' error code."

Reply:
Huh, that's not how it worked in practice for me. We should keep an eye on bugsnag, then, since it might lead to a lot of events if the docs turn out to be right.